docs/docs/configuration/user-implemented-methods.mdx

```diff
@@ -22,6 +22,46 @@ public partial class CarMapper
 
 Whenever Mapperly needs a mapping from `TimeSpan` to `int` inside the `CarMapper` implementation, it will use the provided implementation.
 
+## Automatic user implemented mapping method discovery
+
+By default, user implemented mapping methods are discovered automatically.
```

Comment:
This is the new behavior, correct?

- If `AutoUserMappings = true` (default), then all non-partial methods are considered as user mappings
  - Methods can be manually excluded by setting `[UserMapping(Ignore = true)]`
- If `AutoUserMappings = false`, then users must manually mark methods with `[UserMapping]` for Mapperly to pick them up

correct? I think this is not immediately obvious from the docs.

IMO this paragraph just needs slight adjustments to explain this a little better. What threw me off was the sentence
> To mark methods as user implemented mapping methods the `UserMappingAttribute` has to be applied.

which is only relevant when `AutoUserMappings = false`

Reply:
Adjusted